# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/NeckoParent.h
# Commit: 6e1eba5c88ad
# Full Hash: 6e1eba5c88ad47f0a2a3aa37b4c2cae9f31c2ef3
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 1
# Description:
#   Bug 1569183: Stop doing a proxy lookup to determine whether we're configured to use a proxy (for the proxy_only_if_behind_proxy pref), and instead look at whether we loaded the doc using a proxy. r=mjf,mayhemer,jld
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45289
# ==============================================================================

diff -r c43589fd0f90 -r 6e1eba5c88ad netwerk/ipc/NeckoParent.h
--- a/netwerk/ipc/NeckoParent.h	Tue Sep 17 18:18:33 2019 +0000
+++ b/netwerk/ipc/NeckoParent.h	Wed Sep 18 14:27:42 2019 +0000
@@ -251,13 +251,6 @@
 
   mozilla::ipc::IPCResult RecvEnsureHSTSData(
       EnsureHSTSDataResolver&& aResolver);
-
-  PProxyConfigLookupParent* AllocPProxyConfigLookupParent();
-
-  virtual mozilla::ipc::IPCResult RecvPProxyConfigLookupConstructor(
-      PProxyConfigLookupParent* aActor) override;
-
-  bool DeallocPProxyConfigLookupParent(PProxyConfigLookupParent* aActor);
 };
 
 }  // namespace net